app/feature/feature-login/src/main/kotlin/com/hedvig/android/feature/login/otpinput/OtpInputDestination.kt

```diff
@@ -81,26 +121,25 @@ fun OtpInputScreen(
     },
     modifier = modifier.fillMaxSize(),
   ) { paddingValues ->
-    AnimatedContent(targetState = loadingCode) { loading ->
-      when (loading) {
-        true -> HedvigFullScreenCenterAlignedProgress(show = loadingCode)
-        false -> OtpInputScreenContents(
-          credential,
-          inputValue,
-          onInputChanged,
-          onSubmitCode,
-          networkErrorMessage,
-          onResendCode,
-          loadingResend,
-          onOpenEmailApp,
-          Modifier.padding(paddingValues),
-        )
+    Box(Modifier.fillMaxSize(), propagateMinConstraints = true) {
```

Comment:
Could you tell me why is this better here than AnimatedContent?

Reply:
This was adding an animation between the two cases which wasn't quite necessary here. Practically it meant that when it was loading the content was flickering out and in again quickly if I remember correctly, which was quite jarring.
I just simplified this a bit to just show the loading on top of everything to remove the flickering. In general this one screen is not super well maintained anyway, but I don't think it will stick around for too long regardless.